src/main/java/de/siphalor/mousewheelie/client/inventory/ContainerScreenHelper.java

```diff
@@ -12,121 +12,132 @@
 import net.minecraft.entity.player.PlayerInventory;
 import net.minecraft.item.ItemStack;
 
+import static net.minecraft.client.gui.screen.Screen.hasAltDown;
+
 @SuppressWarnings("WeakerAccess")
 @Environment(EnvType.CLIENT)
 public class ContainerScreenHelper<T extends ContainerScreen<?>> {
-	protected final T screen;
-	protected final ClickHandler clickHandler;
+    protected final T screen;
```

Comment:
You are probably unintentionally changing from tab indentation to space indentation in this file.

Reply:
it fixed in new commit